### PR TITLE
setfacl: add page

### PR DESCRIPTION
**Pull request by touhonoob**

### CODE REVIEW

File: pages/linux/setfacl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# setfacl
+
+> Set file access control lists.
+
+- Modify ACL of file for user with read and write access:
```
Comment (agnivade) [inline pages/linux/setfacl.md]: of "a" file.

File: pages/linux/setfacl.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Remove ACL of a file for an user:
+
+`setfacl -x u:{{username}} {{file}}`
+
+- Remove all ACL entries of a file:
+
+`setfacl -b {{file}}`
+
+- Read ACL of a file:
```
Comment (agnivade) [inline pages/linux/setfacl.md]: I don't think we include another command in one command's page. You can remove this.

Comment (touhonoob) [inline pages/linux/setfacl.md]: fixed

File: pages/linux/setfacl.md (hunk 1 above)
Comment (touhonoob) [inline pages/linux/setfacl.md]: fixed

File: pages/linux/setfacl.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# setfacl
+
+> Set file access control lists.
```
Comment (waldyrious) [inline pages/linux/setfacl.md]: please add the acronym here, as "access control lists (ACL)."

**Comment by waldyrious** [comment]
LGTM. Maybe we can tweak the wording of the example descriptions later on, some of them feel a bit off to me.

**Comment by agnivade** [comment]
:+1:

**Pull request merged**